### PR TITLE
A complete revamp

### PR DESCRIPTION
**Pull request by Remzi1993**
A complete revamp of the code to support wp admin backend settings and allow us much more flexibility.
I made a jQuery Updater for our company I work with, because there were some problems with yours, see: https://core.trac.wordpress.org/ticket/45130

Therefore I decided to make a WordPress plugin, I work at [advanzadirect.nl](https://advanzadirect.nl), I have made two WP plugins for Advanza Direct and [Advanza](https://www.advanza.nl) so I have a little bit of experience with wp development.

I have addressed some issues, jQuery is loaded only on the frontend excluding the wp admin backend and the WP Customizer, because there were some compatibility problems, especially in the WP Customizer (some things didn't work anymore, like editing widgets directly)

Because I have excluded the wp admin and wp customizer, the wp core team has time to move to jQuery 3, see: https://core.trac.wordpress.org/ticket/37110

### CODE REVIEW

**Comment by Remzi1993** [comment]
This is what we internally use at our company: https://github.com/Remzi1993/wp-jquery-updater

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# WP jQuery Updater for WordPress #
+The development and issue tracker is located here
+
+[![GitHub issues](https://img.shields.io/github/issues/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/issues)
+[![GitHub forks](https://img.shields.io/github/forks/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/network)
+[![GitHub stars](https://img.shields.io/github/stars/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/stargazers)
+[![GitHub license](https://img.shields.io/badge/license-GPLv3-blue.svg)](https://github.com/Remzi1993/wp-jquery-updater/blob/master/LICENSE)
```
Comment (Ramoonus) [inline README.md]: oeps?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# WP jQuery Updater for WordPress #
+The development and issue tracker is located here
+
+[![GitHub issues](https://img.shields.io/github/issues/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/issues)
+[![GitHub forks](https://img.shields.io/github/forks/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/network)
+[![GitHub stars](https://img.shields.io/github/stars/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/stargazers)
```
Comment (Ramoonus) [inline README.md]: oeps?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# WP jQuery Updater for WordPress #
+The development and issue tracker is located here
+
+[![GitHub issues](https://img.shields.io/github/issues/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/issues)
+[![GitHub forks](https://img.shields.io/github/forks/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/network)
```
Comment (Ramoonus) [inline README.md]: oeps

File: README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# WP jQuery Updater for WordPress #
+The development and issue tracker is located here
+
+[![GitHub issues](https://img.shields.io/github/issues/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/issues)
```
Comment (Ramoonus) [inline README.md]: oeps

File: README.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# WP jQuery Updater for WordPress #
+The development and issue tracker is located here
```
Comment (Ramoonus) [inline README.md]: improvement: link [here](https://github.com/Ramoonus/jQuery-Updater/)

File: README.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+[![GitHub issues](https://img.shields.io/github/issues/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/issues)
+[![GitHub forks](https://img.shields.io/github/forks/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/network)
+[![GitHub stars](https://img.shields.io/github/stars/Remzi1993/wp-jquery-updater.svg)](https://github.com/Remzi1993/wp-jquery-updater/stargazers)
+[![GitHub license](https://img.shields.io/badge/license-GPLv3-blue.svg)](https://github.com/Remzi1993/wp-jquery-updater/blob/master/LICENSE)
+
+## Bug reports ##
+For bugs, issues use the GitHub [issues](https://github.com/Ramoonus/jQuery-Updater/issues) page.
+If you have any (non bug related) questions, contact me directly
```
Comment (Ramoonus) [inline README.md]: regel kan weg of link naar support pagina

File: README.md (hunk 1 above)
Comment (Remzi1993) [reply README.md]: Ja, dit hoor niet. Sorry mijn fout.

File: README.md (hunk 6 above)
Comment (Remzi1993) [reply README.md]: Oké

**Pull request closed without merging**